packages/react-components/react-theme/stories/Theme/typography/index.stories.mdx

```diff
@@ -1,10 +1,105 @@
-import { Meta, Canvas } from '@storybook/addon-docs';
-import { TypographyStyles } from './ThemeTypography.stories';
+import { Meta, Canvas, Title, Story } from '@storybook/addon-docs';
+import { Text } from '@fluentui/react-components';
+import { Typography } from './ThemeTypography.stories';
+import { Example } from './Example.stories';
+import { Composing } from './Composing.stories';
+import { MixedStyles } from './MixedStyles.stories';
 
-<Meta title="Theme/Typography" />
+<Meta title="Theme/Typography" component={Typography} />
 
-<h1 class="sbdocs-title">Typography</h1>
+<Title>Typography</Title>
+
+Typography style is represented by a set of tokens instead of an individual token. The tokens are used to create and share a consistent look and feel.
+
+<Canvas withSource="none">
+  <Story name="Page" component={Typography} />
+</Canvas>
+
+> **ℹ️ &nbsp;This page guides you on how to fully leverage the tokens to create a consistent typography system.<br />
```

Comment:
I am not sure anybody will ever scroll down here.

Reply:
Moved to the top!